test/Confluent.Kafka.IntegrationTests/Tests/Tests.cs

```diff
@@ -30,13 +30,19 @@ public static IEnumerable<object[]> KafkaParameters()
         {
             if (kafkaParameters == null)
             {
-                var codeBase = typeof(Tests).GetTypeInfo().Assembly.CodeBase;
-                // TODO: Better way to turn Uri into path?
-                var assemblyPath = codeBase.Substring("file://".Length);
+                var assemblyPath = typeof(Tests).GetTypeInfo().Assembly.Location;
                 var assemblyDirectory = Path.GetDirectoryName(assemblyPath);
                 var jsonPath = Path.Combine(assemblyDirectory, "kafka.parameters.json");
-                dynamic json = JsonConvert.DeserializeObject(File.ReadAllText(jsonPath));
-                kafkaParameters = new List<object[]>() { new object[] { json.bootstrapServers.ToString(), json.topic.ToString(), json.partitionedTopic.ToString() } };
+                var json = JObject.Parse(File.ReadAllText(jsonPath));
```

Comment:
With `JsonConvert.DeserializeObject ` we need to have strongly typed object (or dynamic object, like in this case). With `JObject.Parse` we have parsed JSON tree and can access to the object properties without any other object creations. In my opinion, it's better to use `JObject` here instead of `dynamic` object.